core/node/groups.go

```diff
@@ -154,6 +154,7 @@ func LibP2P(bcfg *BuildCfg, cfg *config.Config) fx.Option {
 		fx.Invoke(libp2p.SetupDiscovery(cfg.Discovery.MDNS.Enabled, cfg.Discovery.MDNS.Interval)),
 		fx.Provide(libp2p.ForceReachability(cfg.Internal.Libp2pForceReachability)),
 		fx.Provide(libp2p.StaticRelays(cfg.Swarm.RelayClient.StaticRelays)),
+		fx.Provide(libp2p.HolePunching(cfg.Swarm.EnableHolePunching, cfg.Swarm.RelayService.Enabled.WithDefault(false))),
```

Comment:
@marten-seemann is this `cfg.Swarm.RelayClient.Enabled.WithDefault(false)`?

Reply:
Good catch. Fixed.